ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/facade/TestDefaultAuditor.java

```diff
@@ -109,22 +109,6 @@ public void testAuditFiltered()
         assertThat(timeMeasured).isLessThanOrEqualTo(timeTaken);
     }
 
```

Comment:
This test (testAuditFilteredThrowsException()) should be updated - not removed

To assert that even though an exceptions is thrown, the the audit-method does the correct thing (the new/updated logic).

So this test case should look very similar to testAuditNotFiltered(), except that mockFilter.isWhitelisted() should throw the exception (same as today - line 116)